validator/src/main.rs

```diff
@@ -2331,6 +2357,37 @@ pub fn main() {
     if matches.is_present("no_accounts_db_index_hashing") {
         info!("The accounts hash is only calculated without using the index. --no-accounts-db-index-hashing is deprecated and can be removed from the command line");
     }
+    let retry_rate_ms = value_t_or_exit!(matches, "rpc_send_transaction_retry_ms", u64);
+    let batch_size = value_t_or_exit!(matches, "rpc_send_transaction_batch_size", usize);
+    let batch_send_rate_ms = value_t_or_exit!(matches, "rpc_send_transaction_batch_ms", u64);
+
+    if batch_send_rate_ms > retry_rate_ms || batch_send_rate_ms < 1 {
+        eprintln!(
+            "The specified rpc-send-batch-ms ({}) is invalid, it must be between 1 and the value of rpc-send-retry-ms ({})",
+            batch_send_rate_ms, retry_rate_ms);
+        exit(1);
+    }
+
+    if batch_size <= 0 {
+        eprintln!(
+            "The specified rpc-send-batch-size ({}) is invalid, it must be a positive number.",
+            batch_size
+        );
+        exit(1);
+    }
```

Comment:
these should be enforced with the `clap::Arg::validator()` (use a closure)

Reply:
Done except for checking batch_send_rate_ms > retry_rate_ms which cannot be done with a per arg validator